src/components/K8s.vue

```diff
@@ -28,7 +28,9 @@ const { ipcRenderer } = window.require('electron');
 const fs = window.require('fs');
 const K8s = window.require('./src/k8s-engine/k8s.js');
 const semver = window.require('semver');
-const { exec } = require('child_process');
+const { exec } = window.require('child_process');
+const process = window.require('process');
```

Comment:
global `require` didn't work.

Reply:
I ran into the same problem where `require` didn't work.

@mook-as Do you have any insight on why `require` didn't work for @ericpromislow and we can't use `window.require`?

Reply:
Odd, we do have [`nodeIntegration: true`](https://github.com/mattfarina/rd/blob/a408214fc4f48a31c55cf174854a4692b87b5305/src/window/window.js#L21), so we should be able to use `require` there.  `window.require` isn't going to work because the packers (as in, webpack) will not see them correctly, and therefore not package the dependencies.

Of course, the defence-in-depth alternative is to emit messages over to the main process instead, which is a bit more work.